managing_providers/_topics/embedded_workflows.md

```diff
@@ -181,6 +181,8 @@ Workflows must be authored in Amazon State Languages (ASL) format. As part of au
 
    You are recommended to use a docker.io [access token](https://docs.docker.com/security/for-developers/access-tokens/) so that the token does not expire.
 
```

Comment:
I think we need something to indicate that we are talking about podified/kubernetes here.  This looks like a continuation of the appliance section.

Reply:
@agrare I've added a new heading to each section of 2. to distingush them and clarify what exactly is being dealt with in each section respectively. Thanks!